src/csharp/Grpc.nuspec

```diff
@@ -3,18 +3,17 @@
   <metadata>
     <id>Grpc</id>
     <title>gRPC C#</title>
-    <summary>C# implementation of gRPC - an RPC library and framework</summary>
     <description>C# implementation of gRPC - an RPC library and framework. See project site for more info.</description>
     <version>$version$</version>
-    <authors>Google Inc.</authors>
-    <owners>grpc-packages</owners>
+    <authors>gRPC authors</authors>
+    <owners>gRPC authors</owners>
     <licenseUrl>https://github.com/grpc/grpc/blob/master/LICENSE</licenseUrl>
     <projectUrl>https://github.com/grpc/grpc</projectUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <releaseNotes>Release $version$ of gRPC C#</releaseNotes>
-    <copyright>Copyright 2015, Google Inc.</copyright>
+    <copyright>Copyright 2018, gRPC authors.</copyright>
     <tags>gRPC RPC Protocol HTTP/2</tags>
     <dependencies>
+      <dependency id="Google.Protobuf" version="[3.5, 3.6)" />
```

Comment:
why would we use the version `[3.5, 3.6)`?

Reply:
Well, I do not really know what would be a good range. In the early days of the alpha and beta releases, the protobuf version had to strictly match grpc version, as the generator APIs were changing. The nice side of limiting the range is we do not run into the problem if protobuf generator API or behavior changes. The ugly side if e. g. the user has their own generator that depends on a specific surface version. To me, the former looks more common than the latter, so I'd rather tightened the range reasonably. 

Just `version="3.5` means "version 3.5 or any later future version," which can be problematic if the user of the package upgrades protobuf version too far without also upgrading gRPC.

3.6 would be a reasonable as long as protobuf follows semver. You might be able a more reasonable guess for the upper range than I, from your experience with the gRPC/protobuf mutual dynamics, and user bug reports.